apps/morph_check

```diff
@@ -153,44 +157,12 @@ def parse_args():
 
     return parser.parse_args()
 
-
-# Default check configuration parameters
-_CONFIG = {
-    'checks': {
-        'structural_checks': [
-            'is_single_tree',
-            'has_soma_points',
-            'has_sequential_ids',
-            'has_increasing_ids',
-            'has_valid_soma',
-            'has_valid_neurites',
-        ],
-        'neuron_checks': [
-            'has_basal_dendrite',
-            'has_axon',
-            'has_apical_dendrite',
-            'has_all_nonzero_segment_lengths',
-            'has_all_nonzero_section_lengths',
-            'has_all_nonzero_neurite_radii',
-            'has_nonzero_soma_radius',
-        ]
-    },
-    'options': {
-        'has_nonzero_soma_radius': 0.0,
-        "has_all_nonzero_neurite_radii": 0.007,
-        "has_all_nonzero_segment_lengths": 0.01,
-        "has_all_nonzero_section_lengths": 0.01,
-    },
-    'color': False
-}
-
-
 def main(args):
     '''Run all the checks'''
     _setup_logging(args.debug, args.log_file)
 
     try:
-        config = get_config(args.config, _CONFIG)
+        config = get_config(args.config, os.path.join(CONFIG_PATH, 'morph_check.yaml'))
```

Comment:
morph_check.yaml contains the same config as the previously hard-coded one

Reply:
The 'color' value has a different value. It defaults to 'true' in morph_check.yaml, I don't know if it's a big deal